helm/prometheus-operator-app/templates/alertmanager/alertmanager.yaml

```diff
@@ -10,9 +10,9 @@ metadata:
 spec:
 {{- if .Values.alertmanager.alertmanagerSpec.image }}
 {{- if (.Values.Installation) }}
-  baseImage: "{{ .Values.Installation.V1.Registry.Domain }}/{{ .Values.alertmanager.alertmanagerSpec.image.name }}"
+  image: "{{ .Values.Installation.V1.Registry.Domain }}/{{ .Values.alertmanager.alertmanagerSpec.image.tag }}"
 {{ else }}
-  baseImage: "{{ .Values.image.registry }}/{{ .Values.alertmanager.alertmanagerSpec.image.name }}"
+  image: {{ .Values.alertmanager.alertmanagerSpec.image.repository }}:{{ .Values.alertmanager.alertmanagerSpec.image.tag }}
```

Comment:
The way these are templated needs to be maintained. We define `Values.image.registry` globally in the `values.yaml` to make the registry interchangeable for deployments in china. All references of `Values.<component>.<component>Spec.image.*` in `values.yaml` have been changed as well so line 13 will not work since the definition of the image has changed.

Reply:
So we should always use our image and our registry right ?

Reply:
Yes but the registry needs to be variable so it can be changed to the alibaba registry in china. That's why we had to change all the image templating.